alignak/brok.py

```diff
@@ -107,26 +116,10 @@ def prepare(self):
         # Maybe the brok is a old daemon one or was already prepared
         # if so, the data is already ok
         if hasattr(self, 'prepared') and not self.prepared:
-            if self.use_ujson():
-                # pylint: disable=E1101
-                self.data = ujson.loads(self.data)
-            else:
-                self.data = cPickle.loads(self.data)
+            try:
+                self.data = unserialize(self.data)
+            except AlignakClassLookupException:
+                raise
```

Comment:
We can't log at this level without reworking the logging in Alignak.

Log depend en brok (Brokhandler) and Brok will dep on will if you want to log here. That why I raise the exception and pray for a try except in a higher level :)